mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -124,7 +124,10 @@ class BackgroundTasksManager : BroadcastReceiver() {
             Constants.PREFERENCE_ALARM_CLOCK,
             Constants.PREFERENCE_PHONE_STATE
         )
-        private val VALUE_GETTER_MAP = HashMap<String, (Context) -> String>()
+        private val IGNORED_PACKAGES_FOR_ALARM = listOf(
+            "net.dinglisch.android.taskerm"
+        )
+        private val VALUE_GETTER_MAP = HashMap<String, (Context) -> String?>()
```

Comment:
Probably should make those (up to and including `KNOWN_KEYS`) const

Reply:
`const` does only work with primitives or `String`s.